docs/runtime_usage.md

```diff
@@ -51,23 +51,27 @@ val nullableIntSerializer: KSerializer<Int?> = IntSerializer.nullable
 
 All external serializers (defined by user) are instantiated in a user-specific way. To learn how to write them, see [docs](custom_serializers.md).
 
-### Implicit reflection serializers
+## Reified API
```

Comment:
This document will be completely rewritten soon, so no effort on a real structure here (and quickstart **should not** start with a notion of serializer)

Reply:
Would you do it as a part of this PR or in other?

Reply:
Another PR